hack/cnbi-package-list.yaml

```diff
@@ -0,0 +1,190 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: cnbi-package-list
+  annotations:
+    meteor.zone/pipeline: "true"
+    shower.meteor.zone/button-label: Run code in JupyterHub
+spec:
+  description: Build a notebook image from a package list
+  params:
+    - name: cnbi-package-list-name
+      type: string
+
+  workspaces:
+    - name: package-to-install
+
+  tasks:
+    - name: resolve-dependencies
+      workspaces:
+        - name: pinned-packages
+          workspace: packages-to-install
+      taskRef:
+        name: get-package-versions-list
+      params:
+        - name: CNBI_PACKAGE_LIST
+          value: $(params.cnib-package-list-name)
+    ####### TODO: the rest of the tasks sould be shared between all types of sources
+    - name: get-base-image
+      taskSpec:
+        results:
+          - name: baseImage
+        steps:
+          - stdoutConfig: $(results.baseImage.path)
+            args: # PLACEHOLDER
+              - echo
+              - quay.io/thoth-station/s2i-custom-notebook
+    - name: build-image
+      taskRef:
+        name: buildah-requirements
+        kind: Task
+      runAfter:
+        - resolve-dependencies
+        - get-base-image
+      workspaces:
+        - name: requirements
+          workspace: packages-to-install
+      params:
+        - name: IMAGE
+          value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(context.pipelineRun.name)
+        - name: BASE_IMAGE
+          value: $(tasks.get-base-image.results.baseImage)
+    - name: create-image-stream
+      taskRef:
+        name: openshift-client
+        kind: ClusterTask
+      runAfter:
+        - buildah
+      params:
+        - name: SCRIPT
+          value: |
+            cat <<EOM | oc apply -f -
+            ---
+            kind: ImageStream
+            apiVersion: image.openshift.io/v1
+            metadata:
+              annotations:
+                opendatahub.io/notebook-image-name: $(context.pipelineRun.name)
+              name: $(context.pipelineRun.name)
+              namespace: $(context.pipelineRun.namespace)
+              labels:
+                opendatahub.io/notebook-image: 'true'
+            spec:
+              lookupPolicy:
+                local: true
+              tags:
+                - name: latest
+            EOM
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: buildah-requirements
+spec:
+  params:
+    - name: BASE_IMAGE
+      type: string
+    - description: Reference of the image buildah will produce.
+      name: IMAGE
+      type: string
+    - default: >-
+        registry.redhat.io/rhel8/buildah@sha256:6a68ece207bc5fd8db2dd5cc2d0b53136236fb5178eb5b71eebe5d07a3c33d13
```

Comment:
Do we want/need to pin that much here? (vs pointing to a tag like `8.6`)

Reply:
I don't know ; that's a copy paste from the cnbi-git-repo pipepline.

(Ultimately I'd like to cut most of the tasks and share them between different sources, but I've done simple first)

Reply:
What should we change that to then as a default builder image ?

Reply:
i think pep is suggesting use tag 8.6
`registry.redhat.io/rhel8/buildah:8.6` 
source: https://catalog.redhat.com/software/containers/rhel8/buildah/5dca3d76dd19c71643b226d5?container-tabs=gti